.gitignore

```diff
@@ -5,8 +5,6 @@ node_modules/
 out-tsc/
 debug.log
 npm-debug.log
-src/**/*.js
-terminus-ui/**/*.js
 src/**/*.css
```

Comment:
should we not ignore generated `js` files at all? (instead of having few exceptions)

Reply:
Isn’t that what this change does? Remove any rules to ignore js files?

Reply:
Right... I just want to make sure we're intentional on watching ALL js files, including the ones that're generated from typescript..